src/main/app/claims/models/claim.ts

```diff
@@ -222,6 +234,29 @@ export class Claim {
     return this.settlement && !!this.settlementReachedAt
   }
 
+  private isCCJPaidWithinMonth (): boolean {
+    let futureMonth
+
+    if (this.countyCourtJudgmentIssuedAt) {
```

Comment:
I believe this is going to be removed as part of sujeets story. please check with him.